### PR TITLE
Fixed typo in runTestAtCursor

### PR DESCRIPTION
**Pull request by mikepc**
This fixes a typo that allows runTestAtCursor to work

### CODE REVIEW

**Comment by mikepc** [comment]
Fix for https://github.com/samuelpordeus/vscode-elixir-test/issues/46

File: src/commands/runTestAtCursor.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function handler() {
     const testPathFilter = isUmbrella ? /.*\/(apps\/.*)$/ : /.*\/(test\/.*)$/;
     const terminal = vscode.window.activeTerminal || vscode.window.createTerminal();
     terminal.sendText(
-      `mix test ${openedFilename.match(testPathFilter)[1]}:${cursorLine}`,
+      `mix test ${openedFilename.match(testPathFilter)[1]}: ${cursorLine}`,
```
Comment (ricardoruwer) [inline src/commands/runTestAtCursor.js]: That's weird 🤔
I use this extension in an umbrella app

A space after the colon will not work, e.g.:

```
➜ api(master) mix test apps/api/test/path/file_test.exs:20
==> api
Excluding tags: [:test]
Including tags: [line: "20"]

.

Finished in 0.06 seconds
3 tests, 0 failures, 2 excluded
```

And if we put a space after the colon:

```
➜ api(master) mix test apps/api/test/path/file_test.exs: 20
==> api
Paths given to "mix test" did not match any directory/file: test/api/path/file_test.exs:, 20
```

**Comment by mikepc** [comment]
Odd, modified it on my mac and used the forked version to get past it. I'm
on elixir 1.9 I don't remember mix version.

**Comment by samuelpordeus** [comment]
@mikepc we can see on [Elixir docs](https://hexdocs.pm/mix/Mix.Tasks.Test.html#module-filters) that the way to run a specific line of a test file does not have the extra space at left:
```
mix test test/some/particular/file_test.exs:12
```

It's weird how it helped you to solve this problem 🤔

**Comment by samuelpordeus** [comment]
I'm going to close this PR, but #46 will remain open if you are willing to discuss it more 🙂

**Pull request closed without merging**